acrobat/blocks/prompt-card/prompt-card.css

```diff
@@ -0,0 +1,158 @@
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: "";
+  display: inline-block;
+  height: 18px;
+  min-width: 18px;
+  position: relative;
+  top: 5px;
```

Comment:
Same "apply properties on the parent" comment. Should measurements change, you'd have to make updates in multiple places, whereas if you let the parent handle it, any update will work OOTB.